src/assets/styles/theme.scss

```diff
@@ -34,3 +34,4 @@
 /* Sections */
 @import './sections/image-with-text';
 @import './sections/featured-collection';
+@import './sections/blog-post';
```

Comment:
Nitpick: would you mind placing these in alphabetical order? Personal opinion, I find it easier to compare the list of files in the sidebar with the one in `theme.scss`.